examples/cdk/package.json

```diff
@@ -34,7 +34,8 @@
     "@middy/validator": "^2.5.3",
     "@types/aws-lambda": "^8.10.86",
     "aws-cdk-lib": "2.0.0",
+    "aws-sdk": "^2.1058.0",
```

Comment:
Any specific reason why we are not using the AWS sdk v3? https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/index.html

Reply:
we are using both in the examples to show how to capture both with tracer module